### ISSUE TITLE
Hide printers from dashboard

### ISSUE THREAD

**Issue by taker218** [gdombiak/OctoPod#314]
Hi,

I just updated the iOS App to the latest version and the dashboard is pretty nice.
Would it be possible to add an option to the printers, so you can hide them in the dashboard?

In my case I have 4 printers configured, but they are in fact, just 2 printers.
2 configured printers are for when I'm home, so they have the local IP-addresses configured.
The other 2 have the external IP (with http credentials), so I can monitor them from when I'm not at home and so I don't need to activate the VPN everytime I open the app. They don't show the progress or time left in that case, but that's not a big issue for me.

So, TLDR:
It woul be nice to have the option to hide a printer from the dashboard view :)

Best regards,
Thomas

**Comment by gdombiak** [comment]
Great idea. Just implemented it and will be available in the next release (coming out this week)

![Screen Shot 2020-04-27 at 8 26 21 AM](https://user-images.githubusercontent.com/1791267/80390084-d03b3300-8860-11ea-9da0-df72404c3fc7.png)
![Screen Shot 2020-04-27 at 8 26 04 AM](https://user-images.githubusercontent.com/1791267/80390090-d204f680-8860-11ea-8a1e-03fd3c80c3d8.png)


Thanks,
Gaston

**Comment by taker218** [comment]
awesome, thanks for the quick reply!
